.github/workflows/cmake.yml

```diff
@@ -30,7 +30,7 @@ jobs:
         # on Windows, we rely on vcpkg to pull in dependencies
         shell: bash
         run: |
-          vcpkg install --triplet x64-windows rapidjson
+          vcpkg install rapidjson 'curl[ssl]' --triplet x64-windows
```

Comment:
Do we want to use the static triplet here?

Reply:
Yes, I'd reckon this would be desirable - however, at this point the Github-actions build is more or less only "a check whether stuff compiles", we are not providing any binaries here for download or so. So, I'd argue - at this point, it doesn't matter. I'd therefore leave this to a future change (where e.g. we might offer a binary then).